sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -232,12 +451,64 @@ def from_connection_string(cls, conn_str, **kwargs):
                 :dedent: 4
                 :caption: Create a new instance of the EventHubProducerClient from connection string.
         """
-        constructor_args = cls._from_connection_string(conn_str, **kwargs)
+        constructor_args = cls._from_connection_string(conn_str, eventhub_name=eventhub_name, **kwargs)
         return cls(**constructor_args)
 
+    def send_event(self, event_data, **kwargs):
+        # type: (Union[EventData, AmqpAnnotatedMessage], Any) -> None
+        """
+        Sends an event data.
+        By default the method will blocks until acknowledgement is received or operation times out.
+        If the `EventHubProducerClient` is configured to run in buffered mode, the method will enqueue the event
+        into local buffer and return. The producer will do automatic batching and sending in the background.
```

Comment:
Do we also want to add a sentence here about where/how errors will be raised depending on the `on_error` handler?

Reply:
I added one line of sentences to explain the behavioral differences